src/QirRuntime/test/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 add_subdirectory(FullstateSimulator)
 add_subdirectory(QIR-dynamic)
 add_subdirectory(QIR-static)
+add_subdirectory(Standalone)
```

Comment:
Should we, maybe, create a new higher level folder, something like `samples`, rather than add it to `test`?

Reply:
Yes, that sounds like a good idea. I'll move this to a higher level folder before publishing this PR for merge into `main`.

Reply:
Created a new top level folder called `samples` and added a sub-folder called `StandaloneInputReference` for the input reference.